README.md

```diff
@@ -34,7 +34,15 @@ This Segment [Source function](https://Segment.com/docs/connections/functions/so
 
  <img src="resource/../resources/segment-overview.png" alt="Screenshot of Segment overview showing Fingerprint Pro as a data source">
 
-### ⚠️ Requirements and limitations
+## ⚠️ Requirements and limitations
+
+Please note that this library runs by Segment's custom JavaScript runtime. To test and build locally, this library requires these:
```

Comment:
Most people likely only want to use the source function, not develop it. I would swap the two sections: usage first, development second. :)